src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/Microsoft.Extensions.Logging.Generators.Roslyn4.8.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@ericstj do you see a problem if we upgrade the 

```Xnl
<RoslynApiVersion>$(MicrosoftCodeAnalysisVersion_4_4)</RoslynApiVersion>

Reply:
If we update to that Roslyn version it means we remove the one that targets 4.4.

Updating for the inbox generators in 9.0 could be OK (if we can guarantee that 4.8 is present in all SDK/VS versions where 9.0 is supported).

Updating for the nuget package generators is ok so long as we don't take away any features / perf from folks using roslyn 4.4.  Of course that's moot if noone is using 4.4-4.7.

Somewhere there is a matrix of compiler versions that are supported that could help answer this question.

Reply:
Looks like it doesn't matter in the end. I don't actually need to have code in the generator that's conditionally compiled for Roslyn 4.8. It's only in the test project I need to be sure primary constructors are supported. @tarekgh, is this approach of just upgrading the `RoslynApiVersion` in `Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj` OK?

Reply:
Can you add a new `Microsoft.Extensions.Logging.Generators.Roslyn4.8.Tests.csproj` that target Roslyn 4.8?

Reply:
I had one, but removed it in the last commit since you asked me to revert all changes to the solution file, where I added it. Shouldn't I add it to the solution file if I add a new test project?

Reply:
Yes, if adding a new test project, it will be fine to add it to the solution.